.github/workflows/js-api-build.yml

```
@@ -111,7 +114,8 @@ jobs:
 
             - name: Trigger ADF alpha
               shell: bash
-              if: github.event.schedule == '0 3 * * *'
+              if: github.event.schedule == '0 3 * * *' || github.event_name == 'workflow_dispatch'
```

```suggestion
              if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
```

should be more reliable